tests/test_organization_filters.py

```diff
@@ -86,3 +91,23 @@ def test_dataset_filter_jurisdiction(app: DjangoTestApp):
             'count': 0
         }
     ]
+
+
+@pytest.mark.django_db
+def test_dataset_filter_with_jurisdiction_and_title(app: DjangoTestApp, organizations):
+    resp = app.get("%s?q=1&jurisdiction=Jurisdiction1" % reverse('organization-list'))
+    assert len(resp.context['object_list']) == 1
+    assert resp.context['object_list'][0].pk == organizations[0].pk
+    assert resp.context['selected_jurisdiction'] == "Jurisdiction1"
+    assert resp.context['jurisdictions'] == [
+        {
+            'title': 'Jurisdiction1',
+            'query': "?q=1&jurisdiction=Jurisdiction1",
+            'count': 1
+        },
+        {
+            'title': 'Jurisdiction2',
+            'query': "?q=1&jurisdiction=Jurisdiction2",
+            'count': 0
+        }
+    ]
```

Comment:
Could you also add a test where `q` and `jurisdiction` has special characters like `"<'>\`? I'm not sure, but I think you need to quote user input, I know that values are quoted in templates automatically, but just to be sure.